server/src/test/java/io/crate/testing/DataTypeTesting.java

```diff
@@ -27,22 +27,7 @@
 import com.carrotsearch.randomizedtesting.generators.BiasedNumbers;
 import com.carrotsearch.randomizedtesting.generators.RandomNumbers;
 import com.carrotsearch.randomizedtesting.generators.RandomPicks;
-import io.crate.types.BooleanType;
-import io.crate.types.ByteType;
-import io.crate.types.DataType;
-import io.crate.types.DataTypes;
-import io.crate.types.DoubleType;
-import io.crate.types.FloatType;
-import io.crate.types.GeoPointType;
-import io.crate.types.GeoShapeType;
-import io.crate.types.IntegerType;
-import io.crate.types.IntervalType;
-import io.crate.types.IpType;
-import io.crate.types.LongType;
-import io.crate.types.ObjectType;
-import io.crate.types.ShortType;
-import io.crate.types.StringType;
-import io.crate.types.TimestampType;
+import io.crate.types.*;
```

Comment:
We prefer explicit imports over wildcards, please revert.

Reply:
Done.